app/scripts/modules/core/src/search/global/GlobalSearch.tsx

```diff
@@ -71,7 +71,7 @@ export class GlobalSearch extends React.Component<{}, IGlobalSearchState> {
           .filter(({ results }) => results.length)
           .map(category => ({
             ...category,
-            results: searchRank(category.results).slice(0, 5),
+            results: searchRank(category.results, category.query).slice(0, 5),
```

Comment:
wow. i even re-wrote `searchRank` when i re-wrote all the global search stuff, yet didn't add this query arg.